syntax = "proto2";

message s2c_player_login_complete_notify {
	required uint64 context = 1;	// ±£¡Ù
}